crates/rb-sys/src/stable_api/ruby_2_6.rs

```diff
@@ -88,6 +88,26 @@ impl StableApiDefinition for Definition {
         ptr
     }
 
+    #[inline]
+    unsafe fn rbasic_class(&self, obj: VALUE) -> VALUE {
+        let rbasic = obj as *const crate::RBasic;
+
+        (*rbasic).klass
+    }
+
+    #[inline]
+    unsafe fn rbasic_frozen_p(&self, obj: VALUE) -> bool {
```

Comment:
On consideration, I think we should rename this to `frozen_p` instead and have it Just Work for immediates as well. Thoughts?